src/extension/devtools/panel.ts

```diff
@@ -1,6 +1,9 @@
 import { initDevTools, writeData } from "../../application";
+import { receiveGraphiQLRequests, sendResponseToGraphiQL } from "../../application/Explorer/graphiQLRelay";
 
 (window as any).__DEVTOOLS_APPLICATION__ = {
```

Comment:
If you want to get away from this `as any`, and be explicit about what is allowed in the `__DEVTOOLS_APPLICATION__` object, you can augment the `Window` type to add your own types. There's a good explanation of how to do this [here](https://mariusschulz.com/blog/declaring-global-variables-in-typescript#augmenting-the-window-interface).

Reply:
Keep this as-is for now.